src/PermissionManager.sol

```diff
@@ -250,6 +264,18 @@ contract PermissionManager is IERC1271, Ownable, Pausable {
         emit PermissionContractUpdated(permissionContract, enabled);
     }
 
+    /// @notice Set paymaster should add gas spend or not.
+    ///
+    /// @param paymaster The paymaster contract, potentially spending user assets.
+    /// @param addGasSpend The new setting to add gas spend or not.
+    function setPaymasterGasSpend(address paymaster, bool addGasSpend) external onlyOwner {
+        if (addPaymasterGasSpend[paymaster] == addGasSpend) {
+            revert UnchangedPaymasterGasSpend();
```

Comment:
is this necessary? Maybe just let them call, save an error

Reply:
like how you can call setApprovalForAll twice on the same address, or set an allowance to the same value

Reply:
sounds good, will make sure to be stylistically consistent with `setPermissionContractEnabled` too then in a different PR